doc/rofi-theme.5.markdown

```diff
@@ -667,13 +666,13 @@ The current layout of **rofi** is structured as follow:
 
 
 ```
-> ci is case-indicator
+> ci is the case-indicator
 
 ### Advanced layout
 
 The layout of **rofi** can be tweaked by packing the 'fixed' widgets in a custom structure.
 
-The following widgets names are 'fixed' widgets with functionality:
+The following widgets are 'fixed' with functionality:
```

Comment:
> The following widgets are fixed and provide the following core functionality:

(what is better? core functionality? base functionality?   form (instead of provide) the core functionality of rofi)

To make clear that these are the base widgets used to implement rofi.